### ISSUE TITLE
Bombs do not prime at all only showing activation for 1 second and you can destroy them with guns without detonation

### ISSUE THREAD

**Issue by Kromgar** [tgstation/tgstation#22455]
@Cyberboss 

Ya fucked it up further.

Oh and i got a bomb core from shootign it to destruction.

Im not sure if it updated or not so i may be worng...

but before i could shoot the damn things to the poiunt they blasted. Or if they got destroyed by lava

**Comment by Cyberboss** [comment]
Updates aren't immediate, it's probably fixed. Also it only blows up if the bomb is armed

**Comment by Cyberboss** [comment]
```
<Cyberboss> the servers haven't updated in the past hour have they?
<Haevacht> @seen Mr_Bagil
<Pirou> [1 hour, 46 minutes ago] <Mr_Bagil> AUTOMATIC ANNOUNCEMENT : PR test job finished. PR test will take place next round.
<Haevacht> that's the most recent activity
<Cyberboss> yeah
<Cyberboss> @see Mrs_Sybil
<Cyberboss> @seen Mrs_Sybil
<Pirou> [1 hour, 51 minutes ago] <Mrs_Sybil> AUTOMATIC ANNOUNCEMENT : PR test job started. Merging PR #22287 locally
```

**Comment by Cyberboss** [comment]
Just checked master, everything is fine. This can be closed

**Comment by lzimann** [comment]
Fixed on #22452
